packages/stryker-mutator-specification/src/ConditionalExpressionMutatorSpec.ts

```diff
@@ -0,0 +1,16 @@
+import { expect } from 'chai';
+import ExpectMutation from './ExpectMutation';
+
+export default function ConditionalExpressionMutatorSpec(name: string, expectMutation: ExpectMutation) {
+
+  describe('ConditionalExpressionMutator', () => {
+
+    it('should have name "ConditionalExpression"', () => {
+      expect(name).eq('ConditionalExpression');
+    });
+
+    it('should replace conditional expressions', () => {
+      expectMutation('a < 3? b : c', 'false? b : c');
```

Comment:
I would prefer to add a space between the condition and the `?`

Reply:
That shouldn't matter right? Formatting does not change the intent here.